provider/aws.go

```diff
@@ -364,6 +364,11 @@ func (p *AWSProvider) newChange(action string, endpoint *endpoint.Endpoint) *rou
 		},
 	}
 
+	rec, err := p.Records()
+	if err != nil {
+		log.Errorf("getting records failed: %v", err)
```

Comment:
Does it makes sense to return nil on error or is there a reason to ignore and log this error?
If ignore is correct, then we might just change to Infof log level or do not assign the error return value.